infra/main.bicep

```diff
@@ -29,7 +29,7 @@ param openAiResourceGroupName string = ''
 
 // https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#standard-deployment-model-availability
 @description('Location for the OpenAI resource')
-@allowed([ 'canadaeast', 'northcentralus', 'southcentralus' ])
+@allowed([ 'eastus', 'swedencentral'])
```

Comment:
If we change to Global, we can ditch this. See https://github.com/Azure-Samples/openai-chat-vision-quickstart/blob/main/infra/main.bicep

Reply:
Removed lines 30-37 and switched to the GlobalStnadard sku